architecture/core_objects/routing.adoc

```diff
@@ -119,10 +120,16 @@ file.
 
 ====
 
-Unsecure routes are likely faster to set up, as they are the default configuration, but secure routes offer greater security for information to remain private.
+Unsecure routes are can be faster to set up, as they are the default
```

Comment:
Hmm, i think it should either be "insecure" or "unsecured" (w/ trailing "d").  I'm not sure if the style guide has anything to say about the matter, however...